create_jenkins_job.py

```diff
@@ -114,6 +114,25 @@ def main(argv=None):
             'shell_type': 'Shell',
             'ignore_rmw_default': data['ignore_rmw_default'] | {'rmw_connext_cpp', 'rmw_connext_dynamic_cpp'},
         },
+        'linux-centos': {
+            'label_expression': 'linux_centos',
```

Comment:
As @cottsay and I discussed offline, there's no special capability or change needed on our existing linux agents to run centos containers. These can just use the regular linux nodes and thus no infrastructure changes are required.